aml-pipeline/README.md

```diff
@@ -0,0 +1,71 @@
+## Overview
```

Comment:
Do you mind adding a pointer from the README.md at the root dir to this one with a brief description? Thanks